js/ui/map.js

```diff
@@ -649,6 +658,9 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
      * @param {string} source.type The source type, which must be either one of the core Mapbox GL source types defined in the style specification or a custom type that has been added to the map with {@link Map#addSourceType}.
      * @fires source.add
      * @returns {Map} `this`
+     * @see [Draw GeoJSON points](https://www.mapbox.com/mapbox-gl-js/example/geojson-markers/)
+     * @see [Data-driven circle coloring by ethnicity](https://www.mapbox.com/mapbox-gl-js/example/data-driven-circle-colors/)
```

Comment:
This PR reminded me to go back and rename this example "Style circles using data-driven styling" in https://github.com/mapbox/mapbox-gl-js/commit/30c3f2f8e4ea04a90af02832342a3bcb189a177c